### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by jameszaghini**

### CODE REVIEW

**Comment by jameszaghini** [comment]
A couple of things discussed in Slack are already covered by the existing rules.
- Prefer Swift constructors to legacy ones, e.g. `CGPoint(x: 1, y:1)` over `CGPointMake(1, 1)`
- Forced unwrapping of `IBOutlet's is not mentioned directly but seems acceptable under current guidelines

File: README.markdown [hunk 1]
```diff
@@ -196,24 +197,46 @@ The example above demonstrates the following style guidelines:
...
-  	let message = "Get off the couch " + self.name
-  	print(message)
-  } 
-  
+ init(row: Int, column: Int) {
+   self.row = row
+   self.column = column
+
+   let closure = {
+     println(self.row)
```
Comment (alextrob) [inline README.markdown]: Should use the newer `print()` here.

**Comment by alextrob** [comment]
Looks good to me @jameszaghini. Just one note above. Probably worth updating the other `println`s as well.

**Comment by jameszaghini** [comment]
done :D

**Comment by alextrob** [comment]
:smiley_cat:

**Pull request merged**